utils/model_utils.py

```
@@ -339,18 +344,82 @@
+                batched=True,
+            )
+            training_arguments = TrainingArguments(
+                auto_find_batch_size=True,
+                save_strategy="no",
+                output_dir=output_directory,
+                seed=training_seed,
+                data_seed=data_seed,
+            )
+            trainer = Trainer(
+                model_init=lambda: AutoModelForSequenceClassification.from_pretrained(
+                    model_name,
```

Ruff (0.8.0)

Function definition does not bind loop variable `model_name`

(B023)